src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Crossgen2.sfxproj

```diff
@@ -74,11 +75,35 @@
   <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.SharedFramework.Sdk" />
 
   <Target Name="ResolveReadyToRunCompilers" DependsOnTargets="ResolveRuntimeFilesFromLocalBuild">
+    <PropertyGroup> <!-- Can be removed once runtime repo switches over to 6.0 drop of the SDK repo -->
+      <JitTargetArchComponent>$(TargetArchitecture)</JitTargetArchComponent>
+      <JitTargetOSComponent Condition="'$(TargetOS)' == 'windows'">win</JitTargetOSComponent>
+      <JitTargetOSComponent Condition="'$(TargetOS)' == 'linux'">unix</JitTargetOSComponent>
+      <JitTargetOSComponent Condition="'$(TargetOS)' == 'osx'">osx</JitTargetOSComponent>
+      <JitBuildArchComponent>$(BuildArchitecture)</JitBuildArchComponent>
+      <JitArchSuffix>$(JitTargetArchComponent)_$(JitBuildArchComponent)</JitArchSuffix>
+      <JitFileName>clrjit_$(JitTargetOSComponent)_$(JitArchSuffix)</JitFileName>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(TargetOS)' == 'osx'">
+      <JitFileNameOSX>clrjit_unix_osx_$(JitArchSuffix)</JitFileNameOSX>
+      <JitFileName Condition="Exists($(JitFileNameOSX))">$(JitFileNameOSX)</JitFileName>
+    </PropertyGroup>
```

Comment:
Based on the previous comment, I think this should be removed.

Reply:
Removed in 4th commit. I would still love to hear from @davidwrighton what is the purpose of the file "clrjit_unix_osx_arm64_x64.dll" in my Windows x64 build.

Reply:
It is a temporary JIT library for targeting OSX ARM64, which will be replaced by a runtime switch, see #45501.